pkg/tools/gen/genkcl_jsonschema.go

```diff
@@ -419,7 +420,11 @@ func convertSchemaFromJsonSchema(ctx *convertContext, s *jsonschema.Schema, name
 	if result.HasIndexSignature && result.IndexSignature.validation != nil {
 		result.Validations = append(result.Validations, *result.IndexSignature.validation)
 	}
-	result.property.Name = strcase.ToSnake(result.Name)
+	if ctx.convertToSnake {
```

Comment:
Can we have two options? One convert to snake, second convert to camel, but if no option use original name

Reply:
done.